tests/setupIntegration.js

```
@@ -0,0 +1,66 @@
+global.TextEncoder = require('util').TextEncoder;
+global.TextDecoder = require('util').TextDecoder;
+
+// DO NOT mock fetch - integration tests need real HTTP calls
+// DO NOT mock Supabase credentials - integration tests need real DB
+
+// Suppress console logs but allow them in individual tests if needed
+global.console = {
+  ...console,
+  log: jest.fn((...args) => {
+    // Allow logs that start with emoji (test progress indicators)
+    if (args[0] && typeof args[0] === 'string' && /^[🔌📊✅❌🧹📝🎉⚠️🚀🔄]/.test(args[0])) {
```

Biome (2.1.2)

[error] A character class cannot match a surrogate pair. Add the 'u' unicode flag to match against them.

A surrogate pair forms a single codepoint, but is encoded as a pair of two characters. Without the unicode flag, the regex matches a single surrogate character.
Safe fix: Add unicode u flag to regex


(lint/suspicious/noMisleadingCharacterClass)